README.md

```diff
@@ -2,29 +2,31 @@
 
 ## Secure Font Picker using ArcsJs
 
-This is a prototype implementation of a Picker API for Chrome that would allow
+This is a prototype implementation of a picker API for Chrome built on
+[ArcsJs](https://github.com/project-oak/arcsjs-core) that allows
 permission-less access to fingerprintable resources while being extensible, and
-fingerprint resistant.
+fingerprint resistant. This allows developers to create custom font pickers
+like the one shown below without accessing the full set of local fonts.
 
 ![](https://project-oak.github.io/arcsjs-chromium/demo/explainer/fulldemo.png)
-
 [Live Demo](https://project-oak.github.io/arcsjs-chromium/demo/explainer/fulldemo.html)
 
 The general mechanism for achieving our goal of non-fingerprintability is to
 restrict the code that is processing the list of fonts:
 
-* Only one font can ever be seen by the code at a time
-* The code is pure functional. It can only receive inputs and return outputs. No access to writable global state is permitted.
-* Rendering is done via templates. No access to DOM is permitted.
-* Only a single datum, the selected font, may be returned to the host page.
+- Only one font can ever be seen by the code at a time
+- The code is pure functional. It can only receive inputs and return outputs. No access to writable global state is permitted.
+- Rendering is done via templates. No access to DOM is permitted.
+- Only a single datum, the selected font, may be returned to the host page.
 
 Despite those restrictions, font pickers can be built with rich functionality
 including arbitrary grouping, mixing of local and web fonts, search and sort
 functionality, and even the ability to create and store new per-font metadata
 like a personal rating or favoriting mechanism.
 
 The full system is very flexible, but for the purposes of this tutorial, we will
-only be modifying **Recipes** to customize the system.
+only be modifying [**Recipes**](https://github.com/project-oak/arcsjs-chromium#recipes) 
```

Comment:
I think this is fine but just to mention, we do also have an appendix/section at bottom of the code lab that describes these things. We could link to those here, and link those to these for more -- or keep this. Just raising for re-consideration.